src/components/ApplicationGroup/Details/TriggerView/EnvTriggerView.tsx

```diff
@@ -1305,7 +1316,10 @@ export default function EnvTriggerView({ filteredAppIds, isVirtualEnv }: AppGrou
         }
         const _CDTriggerPromiseList = []
         nodeList.forEach((node, index) => {
-            const ciArtifact = node[materialType].find((artifact) => artifact.isSelected == true)
+            let ciArtifact = null
+            node[materialType].forEach((artifact) => {
```

Comment:
why did we change this?

Reply:
just for readability

Reply:
As discussed. Please rollback this code.